README.md

```diff
@@ -228,6 +229,7 @@ Here are all the samples which are deployed on Microsoft Azure and you can try i
 [meetings-stage-view#cs]:samples/meetings-stage-view/csharp
 [meetings-events#cs]:samples/meetings-events/csharp
 [meetings-events#js]:samples/meetings-events/nodejs
+[meeting-app#cs]:samples/meeting-app/csharp
```

Comment:
@AbhijeetThakur-msft - can we rename this to "meetings-recruitment-app" ? Please make same change to node solution as well.

Reply:
changed